package.json

```diff
@@ -1,8 +1,29 @@
 {
-  "dependencies": {
+  "name": "laminas-ci-matrix-action",
+  "engines": {
+    "node": "^17"
+  },
+  "type": "commonjs",
+  "devDependencies": {
```

Comment:
Ah, what do I have to require?
It actually builds so I havent experienced any problem - but I guess thats because something else is requiring typescript for me?

Reply:
Just add in a dependency to:
```
    "typescript": "~4.6.3"
```
With the `~` constraint instead of `^` since typescript can/does introduce BC breaks in minors.